internal/httpconfig/httpconfig_test.go

```diff
@@ -115,25 +116,42 @@ func TestNTLMProxyInvalidConfigs(t *testing.T) {
 
 	proxyConfig2 := proxyConfig1
 	proxyConfig2.URL, _ = configtypes.NewOptURLAbsoluteFromString("http://fake-proxy")
-	_, err = NewHTTPConfig(proxyConfig2, nil, "", ldlog.NewDisabledLoggers())
+	mockLog2 := ldlogtest.NewMockLog()
+	_, err = NewHTTPConfig(proxyConfig2, nil, "", mockLog2.Loggers)
 	assert.Equal(t, errNTLMProxyAuthWithoutCredentials, err)
+	mockLog2.AssertMessageMatch(t, true, ldlog.Info, "Using proxy server at http://fake-proxy$")
 
 	proxyConfig3 := proxyConfig2
 	proxyConfig3.User = "user"
-	_, err = NewHTTPConfig(proxyConfig3, nil, "", ldlog.NewDisabledLoggers())
+	mockLog3 := ldlogtest.NewMockLog()
+	_, err = NewHTTPConfig(proxyConfig3, nil, "", mockLog3.Loggers)
 	assert.Equal(t, errNTLMProxyAuthWithoutCredentials, err)
+	mockLog3.AssertMessageMatch(t, true, ldlog.Info, "Using proxy server at http://fake-proxy$")
 
 	proxyConfig4 := proxyConfig3
 	proxyConfig4.Password = "pass"
-	_, err = NewHTTPConfig(proxyConfig4, nil, "", ldlog.NewDisabledLoggers())
+	mockLog4 := ldlogtest.NewMockLog()
+	_, err = NewHTTPConfig(proxyConfig4, nil, "", mockLog4.Loggers)
 	assert.NoError(t, err)
+	mockLog4.AssertMessageMatch(t, true, ldlog.Info, "Using proxy server at http://fake-proxy$")
 
 	proxyConfig5 := proxyConfig4
 	helpers.WithTempFile(func(certFileName string) {
 		proxyConfig5.CACertFiles = configtypes.NewOptStringList([]string{certFileName})
-		_, err = NewHTTPConfig(proxyConfig5, nil, "", ldlog.NewDisabledLoggers())
+		mockLog5 := ldlogtest.NewMockLog()
+		_, err = NewHTTPConfig(proxyConfig5, nil, "", mockLog5.Loggers)
 		if assert.Error(t, err) {
 			assert.Contains(t, err.Error(), "invalid CA certificate data")
 		}
+		mockLog5.AssertMessageMatch(t, true, ldlog.Info, "Using proxy server at http://fake-proxy$")
 	})
+
+	proxyConfig6 := proxyConfig4
```

Comment:
Would you mind breaking this out into a new test? Since this test is focused on testing invalid configurations (or at least that's what we named it.)

Reply:
Done!